bdk-ffi/src/bdk.udl

```diff
@@ -374,6 +416,7 @@ dictionary CanonicalTx {
   ChainPosition chain_position;
 };
 
+/// Builds a [`FullScanRequest`].
```

Comment:
These links don't work in the KDocs. I'll send you a screenshot of what they look like, but I was thinking of removing them.